client/src/components/EnquiryModal/EnquiryModal.component.tsx

```diff
@@ -75,15 +75,16 @@ export const EnquiryModal = ({
         <ModalContent w="660px">
           <ModalHeader>
             <Text textStyle="h2" color="secondary.700">
-              {`${agency.shortname.toUpperCase()} Enquiry Form`}
+              {`${(agency?.shortname ?? '').toUpperCase()} Enquiry Form`}
```

Comment:
will there be an extra weird space in front of "Enquiry Form" if `agency` is not defined?

Reply:
Browser happily renders the extra space away
![image](https://user-images.githubusercontent.com/10572368/133364335-2bbb448d-abb1-4964-8aac-7310859a7b6a.png)